### PR TITLE
Turn 13!

### PR DESCRIPTION
**Pull request by uri**
Goodbye random timeout ;) Hello something crazier?

### CODE REVIEW

File: game.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const { Readme } = require('./lib/readme');
 const username = process.env["USER"]
 
 function start() {
+  let fd = fs.openSync("./README.md", 'rs+');
```
Comment (patvice) [inline game.js]: syntax error 😭 😭 

200w

Comment (uri) [reply game.js]: Ugh, shame on me

Comment (uri) [reply game.js]: Original shame can be viewed at beb9185

File: game.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ const { Readme } = require('./lib/readme');
 const username = process.env["USER"]
 
 function start() {
+  Math.random = function() {
+
+  }
```
Comment (kurtfunai) [inline game.js:11]: LOL

**Comment by epixa** [comment]
mother-of-god

**Pull request merged**